v1/pkg/client/docs/Volume.md

```diff
@@ -18,6 +18,7 @@ Name | Type | Description | Notes
 **SubState** | [**VolumeSubState**](VolumeSubState.md) |  | [optional] 
 **Status** | [**VolumeStatus**](VolumeStatus.md) |  | [optional] 
 **Labels** | **map[string]string** | The map of label name to label value for the volume. | [optional] 
+**WWN** | **string** | Serial number of the volume. | [optional] 
```

Comment:
is it World wide name or serial number of the volume? Or is it one & the same?

Reply:
In this case, they are the same.